### ISSUE TITLE
Order of elements not maintained in container when image is used

### ISSUE THREAD

**Issue by nik10-mah** [vvaldersteins/ngx-svg#15]
**Describe the bug**
When an `<svg-image></svg-image>` tag is used along with other elements for example `<svg-line></svg-line>` then the lines get hidden behind the image. As per SVG docs 1.1 to z-index of each element is maintained through the order in which elements are added to container. Element placed 1st will have lower z-index while than the element placed last in the container

**To Reproduce**
Here is the sample code.
```
<svg-container containerId="rootSvg" *ngIf="!loading" [height]="height">
    <svg-image [imageUrl]="storyImg.src" [x]="image.x" [height]="image.height" [width]="image.width" [y]="image.y" [classes]="imgCss"></svg-image>
    <svg-line [borderSize]="line.borderSize" [borderColor]="line.borderColor" id="line1"
          [x0]="line.x0" [y0]="line.y0" [x1]="line.x1" [y1]="line.y1" *ngFor="let line of lines"></svg-line> 
  </svg-container>
```

**Expected behavior**
As per above code  the image should be rendered as first element in svg conatiner and then all of the lines should come.

**Actual behavior**
When you inspect element in chrome the image is being rendered last due to which lines get hidden behind the image.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome

**Comment by vvaldersteins** [comment]
Hi thanks for your request. I'll try to make this one in the next big release v1.3.0 . Will let you know when it is released.

**Comment by vvaldersteins** [comment]
Hi! Just released version 1.3.0 which fixes this. Please let me know if there are any other issues.

**Comment by nik10-mah** [comment]
> Hi! Just released version 1.3.0 which fixes this. Please let me know if there are any other issues.

Thank You so much
